src/helpers/fileHelper.js

```diff
@@ -0,0 +1,8 @@
+export function getFileExtension(fileName) {
```

Comment:
Many files do not have a filename and, hence, no extension! Other files have multiple references ('names') and so this is not a reliable way to do this.

However, the API supplies a `mimetype` field which, together with something like [mime-file-extension](https://www.npmjs.com/package/mime-file-extension) can be used to derive extensions.

The current implementation is good enough for testing purposes but we should most definitely create an issue referring this before this one is merged so we won't forget about it.

This affects a considerable amount of items.

Reply:
The reason for this implementation is that howler.js itself, the chosen player, uses the extension to derive the type. The mimetype does not give accurate/precise enough information, to my knowledge. E.g. audio/mpeg can mean a host of different formats. 
The filetype is only necessary for the audio player. We could hide the player for files that do not have sufficient information available. I would suggest to push that to another MR.